### PR TITLE
Remove JIRA from Community section

### PR DESCRIPTION
**Pull request by oripwk**
Closes https://github.com/apache/airflow-site/issues/258

### CODE REVIEW

**Comment by zhongjiajie** [comment]
FYI, I change you describe to target the exists issue

File: landing-pages/site/content/en/community/_index.html [hunk 1]
```diff
@@ -58,10 +58,9 @@
 
     {{< accordion title="Add a new feature" description="There are two steps required to create feature request for Airflow." logo_path="icons/new-feature-icon.svg" >}}
     <ol class="counter-blue mx-auto">
-        <li>Create account at Airflow JIRA <a href="https://issues.apache.org">[Sign up]</a></li>
+        <li>Create account at Github <a href="https://github.com/">[Sign up]</a></li>
```
Comment (zhongjiajie) [inline landing-pages/site/content/en/community/_index.html:61]:
```suggestion
        <li>Create account at Github <a href="https://github.com">[Sign up]</a></li>
```

File: landing-pages/site/content/en/community/_index.html [hunk 2]
```diff
@@ -73,10 +72,9 @@
 
     {{< accordion title="Fix a bug" description="We don’t like bugs. If you found one please tell us as soon as possible using steps below." logo_path="icons/fix-bug-icon.svg" >}}
     <ol class="counter-blue mx-auto">
-        <li>Create account at Airflow JIRA <a href="https://issues.apache.org">[Sign up]</a></li>
+        <li>Create account at Github <a href="https://github.com/">[Sign up]</a></li>
```
Comment (zhongjiajie) [inline landing-pages/site/content/en/community/_index.html:75]:
```suggestion
        <li>Create account at Github <a href="https://github.com">[Sign up]</a></li>
```

File: landing-pages/site/content/en/community/_index.html [hunk 3]
```diff
@@ -29,7 +29,7 @@
     </ol>
     {{< /accordion >}}
 
-    {{< accordion title="Report a bug" description="Use our [Jira](https://issues.apache.org/jira/projects/AIRFLOW/issues/) to create a ticket." logo_path="icons/bug-icon.svg" >}}
+    {{< accordion title="Report a bug" description="Use our [Github](https://github.com/apache/airflow/issues) to create a ticket." logo_path="icons/bug-icon.svg" >}}
```
Comment (zhongjiajie) [inline landing-pages/site/content/en/community/_index.html:32]: Should we change to github issue icon here?

Comment (oripwk) [reply landing-pages/site/content/en/community/_index.html:32]: There was no JIRA icon here before, just a bug icon, so why there should be a Github icon?

Comment (zhongjiajie) [reply landing-pages/site/content/en/community/_index.html:32]: Screen Shot 2020-04-29 at 16 03 48
I think we use "JIRA icon" named as default bug icon here

Comment (oripwk) [reply landing-pages/site/content/en/community/_index.html:32]: Do you mean the footer?

image

Comment (oripwk) [reply landing-pages/site/content/en/community/_index.html:32]: The bug icon is not a JIRA icon. I don't know where you saw it


image

Comment (zhongjiajie) [reply landing-pages/site/content/en/community/_index.html:32]: Oh, yeah, I make a mistake, could you change the footer too?

Comment (oripwk) [reply landing-pages/site/content/en/community/_index.html:32]: I removed the JIRA icon from the footer, since Github icon is already in the footer

**Pull request merged**